src/components/shared/inputs/FormattedNumberInput.tsx

```diff
@@ -57,57 +62,64 @@ export default function FormattedNumberInput({
   }, [accessoryId])
 
   return (
-    <div
-      style={{
-        display: 'flex',
-        alignItems: 'center',
-        ...style,
-      }}
-    >
-      <InputNumber
-        className={accessory ? 'antd-no-number-handler' : ''}
-        min={min}
-        max={max}
-        style={{ width: '100%' }}
-        value={value !== undefined ? parseFloat(value) : undefined}
-        step={step ?? 1}
-        stringMode={true}
-        placeholder={placeholder}
-        formatter={(val?: string | number | undefined) =>
-          _prefix +
-          (val
-            ? formattedNum(val, {
-                thousandsSeparator,
-                decimalSeparator,
-              })
-            : '') +
-          _suffix
-        }
-        parser={(val?: string) =>
-          parseFloat(
-            (val !== undefined ? val : '0')
-              .replace(new RegExp(thousandsSeparator, 'g'), '')
-              .replace(_prefix, '')
-              .replace(_suffix, '')
-              .split('')
-              .filter(char => allowedValueChars.includes(char))
-              .join('') || '0',
-          )
-        }
-        disabled={disabled}
-        onChange={_value => {
-          if (onChange) onChange(_value?.toString())
-        }}
-      />
-      <div
-        style={{
-          marginLeft: accessoryWidth * -1 - 5,
-          zIndex: 1,
-          fontSize: '.8rem',
-        }}
-      >
-        {accessory && <div id={accessoryId}>{accessory}</div>}
-      </div>
+    <div className="formatted-number-input">
+      {currency ? (
+        <div style={{ marginRight: 5, marginTop: 4 }}>{currency}</div>
```

Comment:
love that you've added in a currency option here. could we replace it with something like this to be more visually consistent? 
```
<FormattedNumberInput
    ...
    accessory={
        <InputAccessoryButton
              content={currencyName(currency)}
        />}
/>
```

Reply:
Ahh yep cool forgot about that component, thanks! 

<img width="263" alt="Screen Shot 2022-01-08 at 10 20 59 am" src="https://user-images.githubusercontent.com/96150256/148623243-c98dc235-a458-4342-a620-2e71a385edf7.png">